File: backend/Domain/Models/Nutrients.cs
using System.Text.Json.Serialization;
using Domain.DataBaseAccess;

namespace Domain.Models;

public class Nutrients : ITableDataBase
{
    public long? Id { get; set; }
    [JsonPropertyName("menu_item_id")] public long? MenuItemId { get; set; }
    public int? Kilocalories { get; init; }
    public int? Proteins { get; init; }
    public int? Fats { get; init; }
    public int? Carbohydrates { get; init; }

    public Nutrients(){}
    public Nutrients(long? id, int? kilocalories, int? proteins, int? fats, int? carbohydrates)
    {
        Id = id;
        Kilocalories = kilocalories;
        Proteins = proteins;
        Fats = fats;
        Carbohydrates = carbohydrates;
    }
}

File: backend/API/PY_bots/Business/Model/Nutrients.py
from pydantic import BaseModel
from typing import Optional


class Nutrients(BaseModel):
    id: Optional[int]
    menu_item_id: int
    kilocalories: Optional[int]
    proteins: Optional[int]
    fats: Optional[int]
    carbohydrates: Optional[int]

    def __init__(self,
                 menu_item_id: int,
                 kilocalories: Optional[int] = None,
                 proteins: Optional[int] = None,
                 fats: Optional[int] = None,
                 carbohydrates: Optional[int] = None,
                 id: Optional[int] = None):
        super().__init__(
            kilocalories=kilocalories,
            proteins=proteins,
            fats=fats,
            carbohydrates=carbohydrates,
            menu_item_id=menu_item_id,
            id=id
        )